### DISCUSSION TITLE
npm ERR! cb() never called!

### DISCUSSION THREAD

**Discussion by GitHubTuna** [npm/feedback#533] [CLI] [answered]
Anyone else fighting to get passed this with nearly all NPM commands?

**Comment by MylesBorins** [comment, marked as answer]
Hey @GitHubTuna . This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!
